scripts/us_fema/national_risk_index/README.md

```diff
@@ -0,0 +1,66 @@
+# Importing FEMA National Risk Index (NRI) Data
+
+This directory imports the [FEMA NRI](https://hazards.fema.gov/nri/) dataset
+into Data Commons. The study includes relative measures of risks from 18 hazards
+to the US at the county and tract level, as well as data on individual hazards
+and their estimated risk to those affected.
+
+Source data can be downloaded from this [download page](
+https://hazards.fema.gov/nri/Content/StaticDocuments/DataDownload//NRI_Table_CensusTracts/NRI_Table_CensusTracts.zip
+).
+
+# Instructions
+
+## Generating Artifacts
+
+To generate the schema and the TMCF for the import, run
+```
+python3 generate_schema_and_tmcf.py
+```
+
+To clean the source data, run
+```
```

Comment:
It seems process_data.py relies on csv_columns.json which gets produced by earlier script?  Indicate that.  Also, it might be useful to wrap both these in a single generate.sh script (esp based on comment below we will also download the zip instead of checking it in)...

Reply:
noted this relation in the new sh/generate.sh script. I am looking into downloading the file now...